scipy/optimize/_linprog.py

```diff
@@ -460,6 +460,9 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
     >>> x1_bounds = (-3, None)
     >>> from scipy.optimize import linprog
     >>> res = linprog(c, A_ub=A, b_ub=b, bounds=[x0_bounds, x1_bounds])
+
+    Note that the default method for `linprog` is 'interior-point', which is
+    approximate by nature.
```

Comment:
Here you will need to insert a blank line after line 465.

Reply:
Added. Hope this fixed it.